localization/ci.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<localizationPack name="Côte d'Ivoire" version="1.0">
```

Comment:
```suggestion
<localizationPack name="Ivory Coast" version="1.0">
```

Reply:
OK

Reply:
@SylNicky, thanks for the comment! You can click on the 'Commit suggestion' buttons to take those three modifications into account. After that, the pull request will be ready. ;-)

Reply:
Done

Reply:
Thank you very much!